usaspending_api/common/tests/integration/test_decorators.py

```diff
@@ -0,0 +1,60 @@
+# Stdlib imports
```

Comment:
It'd also be good to have a inverse test with a pg_sleep and no decorator, and prove that the time it took to run was >= to the pg_sleep amount.